docs/reference/mapping/types/core-types.asciidoc

```diff
@@ -379,7 +379,9 @@ defaults to `true` or to the parent `object` type setting.
 |`ignore_malformed` |Ignored a malformed number. Defaults to `false`.
 
 |`numeric_resolution` |The unit to use when passed in a numeric values. Possible
```

Comment:
sorry, I accidentally looked in an older branch for the timestamp before you removed the docs for the lockdown, makes sense to stay consistent.. just changed it